src/components/book/Book.js

```diff
@@ -1,21 +1,49 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { removeBook } from '../../redux/books/books';
+
+const Book = (props) => {
+  const dispatch = useDispatch();
+  const { book } = props;
+  const { id, title, author } = book;
+
+  const handleRemoveBook = (book) => {
+    dispatch(removeBook(book.id));
+  };
```

Comment:
- [ ] Kindly notice that the remove action is not working you need to remove the `book` parameter from `handleRemoveBook` to make remove work.

```javascript
const handleRemoveBook = () => {
    dispatch(removeBook(book.id));
};
```